application/src/main/java/com/softserve/lv460/application/controller/FeatureController.java

```diff
@@ -0,0 +1,39 @@
+package com.softserve.lv460.application.controller;
+
+import com.softserve.lv460.application.dto.feature.FeatureRequest;
+import com.softserve.lv460.application.dto.feature.FeatureResponse;
+import com.softserve.lv460.application.service.FeatureService;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/feature")
+@CrossOrigin
+public class FeatureController {
+
+  private FeatureService featureService;
+
+  public FeatureController(FeatureService featureService) {
```

Comment:
Since we are already using lombok's AllArgsConstructor, use it here as well for consistency